.travis.yml

```diff
@@ -45,16 +45,13 @@ before_install:
   - xdebug-disable
 
 install:
-  - composer install
   - |
    if [ "${SYMFONY_VERSION}" != "" ]; then
       composer config --unset platform.php
-      composer require \
-        symfony/console:${SYMFONY_VERSION} \
-        symfony/process:${SYMFONY_VERSION} \
-        symfony/finder:${SYMFONY_VERSION} \
-        symfony/yaml:${SYMFONY_VERSION}
+      composer require --no-update symfony/force-lowest:$SYMFONY_VERSION
+      composer require --no-update symfony/lts:$(echo $SYMFONY_VERSION | grep -o 'v[0-9]\+') || true
```

Comment:
the 4th version of the symfony is not a LTS and also https://packagist.org/packages/symfony/lts doesn't have `v4`. Did you check your approach locally?

p.s. tbh we've already tried to use `symfony/lts`